public/locales/fr/translation.json

```diff
@@ -555,7 +555,12 @@
         "fileImportError": "Échec du chargement du fichier.",
         "maxErrorsCount": "(seulement les {{maxCount}} premiers résultats)",
         "errorsCount_one": "{{count}} erreur:",
-        "errorsCount_other": "{{count}} erreurs:"
+        "errorsCount_other": "{{count}} erreurs:",
+        "placeholdertitle": "Importer votre premier fichier",
+        "placeholderbody": "Une fois que vous ayez importé votre premier fichier csv ou xls, vos données vont être affiché dans un tableau intéractif.",
```

Comment:
typo: données vont être affich**ées** and int**E**ractif (without **é**)

Reply:
DONE